middlewares/auth.js

```diff
@@ -0,0 +1,43 @@
+const passport = require("passport");
+const passportJWT = require("passport-jwt");
+const User = require("../service/schemas/user");
+require("dotenv").config();
+const secret = process.env.SECRET;
```

Comment:
```suggestion
const secret = process.env.SECRET;
const SECRET = process.env.SECRET;
```
Ta linijka jest w porządku, natomiast  jeśli do stałej przypiszemy wartość która się nie zmienia, to pisze się je `SECRET` .  Pozwala to szybciej zrozumieć że ona jest na swój sposób wyjątkowa, w tym wypadku jest konfiguracją przekazaną z .env.